### DISCUSSION TITLE
Am I skilled enough?

### DISCUSSION THREAD

**Discussion by coderDruti** [dipy/dipy#2758] [#GSoC/GSoD]
![Screenshot 2023-03-24 160627](https://user-images.githubusercontent.com/113259277/227500048-b69a61db-9657-47d5-98d2-d676f8edba90.png)
Hello, I am Druti from India. I am a CSE undergraduate, studying in 2nd year of BTech in CSE. I have completed a short course on data science and machine learning and have a basic knowledge of Python, Java and C/C++. The reason for this post is that I want to know whether I am skilled enough for the project, attached herein. Also, if I am not skilled enough please inform me of what areas I can improve upon or learn new skills. Thank you for reading.

**Comment by skoudoro** [comment]
Hi @coderDruti,

I recommend you to read this blog post: https://levelup.gitconnected.com/my-journey-guide-for-google-summer-of-code-7b03e1007c07

You will quickly know if you are skilled enough or not.

Enjoy the reading.
